apps/protocol/src/app/context/SelectedSaveVersionProvider.tsx

```diff
@@ -40,7 +40,7 @@ export const SelectedSaveVersionProvider: FC = ({ children }) => {
       account: walletAddress ?? '',
       include: !!(v1Address && walletAddress),
     },
-    skip: !v1Address,
+    skip: !v1Address || !walletAddress,
```

Comment:
I saw this query erroring in a user screenshot, seems like this check is needed (so it skips if there's no account)